include/formbase.php

```diff
@@ -201,9 +201,23 @@ function getPostToForm($ignore='', $isRegularExpression=false)
 
 function getGetToForm($ignore='', $usePostAsAuthority = false)
 {
+    global $log;
 	$fields = '';
-	foreach ($_GET as $key=>$value)
+	foreach ($_GET as $key => $value)
 	{
+	    if(is_array($key)) {
+            if(!empty($key)) {
+                $log->warn('$key must be a string');
+            }
+	        continue;
+        }
+
+        if (is_array($value)) {
+            if(!empty($value)) {
+                $log->warn('$value must be a string');
```

Comment:
I don't really understand this log message. the fact when a $value is a non-empty array is not implicated it to a string. any thing else also could not being a string...

Reply:
could you use simply `is_string` function OR change the log message to  what is the problem..